sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,29 +1,41 @@
 # Release History
 
-## 2.0.0-beta.1 (Unreleased)
+## 2.0.0-beta.1 (2021-03-23)
 
 This release continues with the changes from `1.2.4` and `1.2.4-beta.1`.
 
+### Breaking changes
+
 - The `getToken` methods will now never return `null`. If a token is not available, we will return a rejected promise.
-- `DefaultAzureCredential`'s implementation for browsers was simplified to throw a simple error instead of trying credentials that were already not supported for the browser.
-- Breaking Change: `InteractiveBrowserCredential` for the browser now requires the client ID to be provided.
-- Documentation was added to elaborate on how to configure an AAD application to support `InteractiveBrowserCredential`.
-- Replaced the use of the 'express' module with a Node-native http server, shrinking the resulting identity module considerably
-- Updated `@azure/msal-node-extensions` to [1.0.0-alpha.6](https://www.npmjs.com/package/@azure/msal-node-extensions/v/1.0.0-alpha.6).
+- `InteractiveBrowserCredential` for the browser now requires a client ID to be provided.
+- `InteractiveBrowserCredential` now only has `loginStyle` and `flow` in the optional parameters when the credential is bundled for browsers. This reflects the intended behavior, since the `loginStyle` has no effect on NodeJS.
```

Comment:
This is not clear to me....

> `InteractiveBrowserCredential` now only has `loginStyle` and `flow` in the optional parameters when the credential is bundled for browsers.

This tells me that there was more than loginStyle and flow before, but we don't mention what that is

> This reflects the intended behavior, since the `loginStyle` has no effect on NodeJS

This sentence would have made sense if we were saying that `loginStyle` is no longer available when using in node. But that is not what the previous sentence said

Reply:
`loginStyle` was available as a parameter on node but didn't do anything 🤔

Reply:
What about this?

> While the NodeJS version of `InteractiveBrowserCredential` allowed the `loginStyle` and `flow` optional properties to be passed through, these are only useful in the browser version of the credential. For that purpose, we've removed these parameters from the Node JS version of this credential.

Reply:
Same for `flow`? Then this should say

> The `loginStyle` and `flow` options to the constructor for `InteractiveBrowserCredential` will now be applicable only when used in browser as this does not apply to node.js